tailwind.config.js

```diff
@@ -65,13 +65,9 @@ module.exports = {
   },
   plugins: [
     plugin(({ addVariant, addUtilities, variants }) => {
-      // imitation of the twin.macro svg: variant. svg:text-green-500 puts green
-      // on an SVG that's an immediate child of the element
-      addVariant('svg', '& > svg')
```

Comment:
I think when we added this, you couldn't do it inline as an arbitrary value. Now you can.